Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: all docs test itest
+
+docs: .taskproc/bin/activate
+	.taskproc/bin/tox -e docs
+
+test: .taskproc/bin/activate
```

Comment:
You should also make sure pre-commit is installed, doing it as part of `make test` is a good idea - either here or from within tox.